src/Uno.UI.RemoteControl/RemoteControlClient.cs

```diff
@@ -76,6 +76,8 @@ private RemoteControlClient(Type appType)
 			_ = StartConnection();
 		}
 
+		public bool IsConnected => _webSocket is not null;
```

Comment:
I see once the connection is made, we never close it, so this should never be `null` once the connection is established.

Reply:
At the moment, I'm using this as a way to detect if a connection has been established, and so closing isn't a concern.
The connection is closed when a debug session finishes.
If something forces the connection to break, the current behavior/requirement is to restart the debug session to use the functionality requiring RemoteControl.
If (and when) needed, we can add the capability to restart the connection without restarting the debug session later.

Reply:
There's a similar change in https://github.com/unoplatform/uno/pull/12846, (https://github.com/unoplatform/uno/pull/12846/files#diff-0a4e1397a5062960f20dea694ab5858fafa2ebd5973737aca619d8869f62814aR90), with `WaitForConnection`. Do you need to asynchronously wait for client to be connected, or is it for something else?

Reply:
Looks like we're both trying to address the same issue. 
I wanted a quick way to check when the RCClient had a connection before I started sending lots of messages through it.
I was looking for a way to make minimal changes to RCClient but if there are separate bigger changes happening there I can wait for them.